ci/parse-trivy-results.py

```diff
@@ -0,0 +1,121 @@
+"""Script to parse results from Trivy scan.
+
+1) Reads in results from two files (json and table format) generated by Trivy.
+2) Creates a GitHub issue in the given repo if any vulnerabilities were found.
+3) Fails the CI build if any vulerabilities of HIGH/CRITICAL severity exist.
+
+Both the json and table Trivy results must be created prior to running 
+this script. These can be generated with:
+    trivy image --format json --light --no-progress -o <OUTPUT_NAME> 
+        gcr.io/cre-tools/ts-bridge
+    trivy image --light  --no-progress -o <OUTPUT_NAME>
+        gcr.io/cre-tools/ts-bridge
+
+Usage:
+    python3 parse-trivy-results.py <TRIVY_FILE> <COMMIT_ID> <BUILD_ID> <FULL_TAG> <REPO_NAME> <TOKEN_FILE>
```

Comment:
Add example to clarify format of TRIVY_FILE, COMMIT_ID, REPO_NAME, FULL_TAG and TOKEN_FILE

Reply:
This has been replaced by absl.flags